Issue #498: Implement a simple print function for forecast objects. (#592)
epiforecasts/scoringutils: NEWS.md

- Added unit tests for `interval_coverage_quantile()` and `interval_coverage_dev_quantile()` in order to make sure that the functions provide the correct warnings when insufficient quantiles are provided.
- Documentation pkgdown pages are now created both for the stable and dev versions.
- Output columns for pairwise comparisons have been renamed to contain the name of the metric used for comparing.

> - Added `print()` on a `forecast` object to print out diagnostic messages. 

Reviewer: ```suggestion
- Added a method for `print()` that prints out additional information for `forecast` objects. 
```
